modules/installation-installing-bare-metal.adoc

```diff
@@ -36,12 +36,12 @@ endif::restricted[]
 . Monitor the bootstrap process:
 +
 ----
-$ ./openshift-install --dir=<installation_directory> wait-for bootstrap-complete \ <1>
-    --log-level=info <2>
-
-INFO Waiting up to 30m0s for the Kubernetes API at https://api.test.example.com:6443...
-INFO API v1.14.6+c4799753c up
-INFO Waiting up to 30m0s for the bootstrap-complete event...
+$ ./openshift-install --dir=/root/manual_installation/installation_directory wait-for bootstrap-complete \
+    --log-level=info
+INFO Waiting up to 30m0s for the Kubernetes API at https://api.m3559ocp.lnxne.boe:6443...
```

Comment:
Just wondering if this should stay `api.test.example.com` instead of `api.m3559ocp.lnxne.boe`

Reply:
Are we allowed to just change the outputs, trading their "realism" for better presentation? If yes, I'll change this and in a couple of other outputs.

Reply:
Absolutely, I do it all the time. Better to present a generic example instead of the exact machine/cluster info from where we actually ran the tests.

Reply:
Great, will keep it in mind. Changed to `api.test.example.com` and changed in a few other places.